@navikt/core/react/src/form/file-upload/parts/dropzone/Dropzone.tsx

```diff
@@ -76,12 +78,14 @@ const Dropzone = forwardRef<HTMLInputElement, FileUploadDropzoneProps>(
             {description}
           </BodyShort>
         )}
+        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
         <div
+          className="navds-dropzone__area"
+          onDragEnter={dropzoneCtx.onDragEnter}
           onDragOver={dropzoneCtx.onDragOver}
           onDragLeave={dropzoneCtx.onDragLeave}
-          onDragEnd={dropzoneCtx.onDragEnd}
```

Comment:
onDragEnd trengs ikke, da det er for når man drar elementet den er satt på (dropsonen).